vxsandbox/resources/metrics.py

```diff
@@ -75,10 +78,35 @@ def from_command(cls, command):
 
 
 class MetricsResource(SandboxResource):
-    """Resource that provides metric storing."""
+    """Resource that provides metric storing.
+
+    TODO: Describe how to set metrics_prefix.
+    #         return "%scampaigns.%s.stores.%s." % (
+    :param dict metrics_prefix:
+        metrics prefix configuration.
+
+    """
+    @inlineCallbacks
+    def setup(self):
+        prefix = self.config.get('metrics_prefix', None)
+        if prefix is None:
+            raise ConfigError("Metrics key given does not exist")
```

Comment:
This should probably say something like "metrics_prefix config parameter not supplied", since it's not that the metric key doesn't exists, it wasn't given in the config.